app/models/feed.rb

```diff
@@ -211,6 +219,12 @@ def mime_type
   def copy_media(force = false)
     feed_images.each { |i| i.copy_media(force) }
     itunes_images.each { |i| i.copy_media(force) }
+
+    # remove old feed rss
+    if !previously_new_record? && (slug_previously_changed? || file_name_previously_changed?)
```

Comment:
Terrible idea #2 - after a controller saves a Feed and calls `.copy_media`, check if we just changed one of these fields.

Removing the old RSS file is nice, but not critical.  So I think this is good enough?